core/org.wso2.carbon.core/src/main/java/org/wso2/carbon/core/util/KeyStoreManager.java

```diff
@@ -423,8 +545,43 @@ public KeyStore getInternalKeyStore() throws Exception {
             }
             return internalKeyStore;
         } else {
-            throw new CarbonException("Permission denied for accessing internal key store. The internal key store is " +
-                    "available only for the super tenant.");
+            throw new CarbonException(String.format(PERMISSION_DENIED_ERROR, "internal key store", "internal key store"));
+        }
+    }
+
+    /**
+     * Load the trust store, this is allowed only for the super tenant.
+     *
+     * @return Trust store object.
+     * @throws SecurityException If an error occurs while loading the trust store.
+     * @throws CarbonException Carbon Exception when trying to call this method from a tenant other
+     *                   than tenant 0
```

Comment:
Shall we use word super tenant here instead of tenant 0

Reply:
Fixed with https://github.com/wso2/carbon-kernel/pull/4091/commits/767822c4dacfe7deb0d9d42282a588835bdd690e